extensions/amp-analytics/0.1/iframe-transport.js

```diff
@@ -180,12 +188,13 @@ export class IframeTransport {
   sendRequest(event) {
     const frameData = IframeTransport.getFrameData(this.type_);
     dev().assert(frameData, 'Trying to send message to non-existent frame');
-    dev().assert(frameData.queue, 'Event queue is missing for ' + this.id_);
+    dev().assert(frameData.queue,
+        'Event queue is missing for ' + this.getCreativeId());
```

Comment:
the error message here is not very clear. Since different `iframeTransport` instance can reuse the same `creativeId`, the `creativeId` plus the `type` should be used to identify the error.

Reply:
Fixed